docs/api/devtime_dependencies.md

```
@@ -0,0 +1,275 @@
+
+```
+
+### Example: Adding a development-time dependency with State Management
+
+This example demonstrates how to use DevTimeDependencies with the State for dependency injection in a Fiber application.
+
+```go
+package main
+
+import (
+	"context"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)